>From 54840b1e36887d894a41f7e8c9ee6a0f7bcefd40 Mon Sep 17 00:00:00 2001
From: Cyril Brulebois <kibi@debian.org>
Date: Tue, 18 Feb 2014 20:20:27 +0300
Subject: [PATCH 3/4] deb-reversion: Mention when a changelog file is expected
 and missing.

A very common case is udeb packages.
---
 scripts/deb-reversion.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/deb-reversion.sh b/scripts/deb-reversion.sh
index 2a17e56..2ef7edd 100755
--- a/scripts/deb-reversion.sh
+++ b/scripts/deb-reversion.sh
@@ -173,7 +173,7 @@ change_version()
     [ -f usr/share/doc/${PACKAGE}/$i ] \
       && LOGFILE=usr/share/doc/${PACKAGE}/$i
   done
-  [ -z "$LOGFILE" ] && return 1
+  [ -z "$LOGFILE" ] && { echo "changelog file not found"; return 1; }
   mkdir -p debian
   zcat $LOGFILE > debian/changelog
   shift
-- 
1.7.10.4

